examples/goldilocks_models/find_goldilocks_models.cc

```diff
@@ -2275,13 +2378,18 @@ int findGoldilocksModels(int argc, char* argv[]) {
 
           // Get file name of initial seed
           string init_file_pass_in;
-          getInitFileName(&init_file_pass_in, init_file, iter, sample_idx,
-                          is_get_nominal,
+          int total_sample_num = N_sample_sl*N_sample_gi;
```

Comment:
Should this be `N_sample`? (removing `total_sample_num`)

Reply:
Also, it's probably a bug that `N_sample_tr` was not included here. (so the previous experiment didn't perform as well as it should have)

Reply:
Resolved.